src/models/room.ts

```diff
@@ -2141,9 +2148,19 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
             };
         }
 
-        // We've exhausted all scenarios, can safely assume that this event should live in the room timeline only
+        const unsigned = event.getUnsigned();
+        if (typeof unsigned["org.matrix.msc4023.thread_id"] === "string") {
```

Comment:
Are we still using the `UnstableValue` abstraction or scrapping it? I'm personally fine with this!

Reply:
Good call, forgot about it